index.html

```diff
@@ -270,6 +271,17 @@ <h3>Conceptual Framework</h3>
           </figcaption>
         </figure>
 
+        <p class="note"
+           title="Status information is about the verifiable credential">
+Ultimately, the status information associated with a particular
+<a>verifiable credential</a> is about the verifiable credential itself and
+might not always be about the underlying credential, such as
+an educational degree. For example, in the case of an educational degree,
+it is possible for the associated <a>verifiable credential</a> to be revoked
+because the mechanism used to create the digital signature has been compromised,
+while the underlying educational degree remains valid.
```

Comment:
```suggestion
The status information associated with a particular <a>verifiable credential</a>
is about the <a>verifiable credential</a> itself and might not apply to any
underlying or backing <a>credential</a>, such as an educational degree. For
example, in the case of such an educational degree, it is possible for a
<a>verifiable credential</a> to be revoked because the mechanism used to
create its digital signature has been compromised, while the backing educational
degree remains valid.
```